pycparser/c_parser.py

```diff
@@ -310,6 +310,11 @@ def _fix_decl_name_type(self, decl, typename):
         decl.name = type.declname
         type.quals = decl.quals[:]
 
+        for tn in typename:
+            if isinstance(tn, c_ast.Alignas):
+                type.align = tn
+                typename.remove(tn)
```

Comment:
Don't modify a list you're iterating.

Reply:
Also, if we allow a list of alignment specifiers, does it show here or do we only overwrite them?

Reply:
Fixed.